src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Migration Overview
+summary: Learn how to migrate your database to a CockroachDB cluster.
+toc: true
+docs_area: migrate
+---
+
+The MOLT (Migrate Off Legacy Technology) toolkit enables safe, minimal-downtime database migrations to CockroachDB. MOLT combines schema transformation, distributed data load, continuous replication, and row-level validation into a highly configurable workflow that adapts to diverse production environments.
+
+This page has an overview of the following:
+
+- Overall [migration flow](#migration-flow)
+- [MOLT tools](#molt-tools)
+- Supported [migration and failback modes](#migration-modes)
+
+## Migration flow
+
+{{site.data.alerts.callout_success}}
+Before you begin the migration, review [Migration Strategy]({% link molt/migration-strategy.md %}).
+{{site.data.alerts.end}}
+
+A migration to CockroachDB generally follows this sequence:
+
+<div style="text-align: center;">
+<img src="{{ 'images/molt/migration_flow.svg' | relative_url }}" alt="MOLT tooling overview" style="max-width:100%" />
+</div><br>
+
+1. Prepare the source database: Configure users, permissions, and replication settings as needed.
+1. Convert the source schema: Use the [Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) to generate CockroachDB-compatible [DDL]({% link {{ site.current_cloud_version }}/sql-statements.md %}#data-definition-statements). Apply the converted schema to the target database. Drop constraints and indexes to facilitate data load.
+1. Load data into CockroachDB: Use [MOLT Fetch]({% link molt/molt-fetch.md %}) to bulk-ingest your source data.
+1. (Optional) Verify consistency before replication: Use [MOLT Verify]({% link molt/molt-verify.md %}) to confirm that the data loaded into CockroachDB is consistent with the source.
+1. Replicate ongoing changes: Enable continuous replication with [MOLT Fetch]({% link {{ page.version.version }}/molt-fetch.md %}#replicate-changes) to keep CockroachDB in sync with the source.
+1. Verify consistency before cutover: Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to confirm that the CockroachDB data is consistent with the source.
+1. Finalize target schema: Recreate indexes or constraints on CockroachDB that you previously dropped to facilitate data load.
```

Comment:
The ordering of this can be dependent. Sometimes it would be better to do it after the initial data load is done so that way cutover is easier. Otherwise if you hadd the indexes after you start downtime, it could be alot more expensive.